app/scripts/models/auth_brokers/first-run.js

```diff
@@ -18,6 +18,7 @@ define([
     _iframeCommands: {
        LOADED: 'loaded',
        LOGIN: 'login',
+       VERIFY_SIGNUP: 'verify_signup',
```

Comment:
From @jpetto in IRC:
> [15:16:30]  <jpetto>	ah - wasn't sure if that question was for me or not ;)
[15:16:56]  <jpetto>	it doesn't quite fit with the other message names
[15:17:16]  <jpetto>	the others describe what happened - resize, verification_complete, login
[15:17:33]  <jpetto>	so yeah, i'd say must_verify_signup is better
[15:18:10]  <jpetto>	or even signup_unconfirmed
[15:18:18]  <jpetto>	or signup_unverified